WebServer/testing/run_tests.py

```diff
@@ -0,0 +1,23 @@
+import os
+import subprocess as sp
+import unittest
+
+
+
+def run_unit_tests():
+    os.chdir(os.getcwd() + "/../microservices/db/unittest")
+    sp.call("python -m unittest discover -s . -p '*_test.py'",shell=True)
+    os.chdir(os.getcwd() + "/../../dispatcher/unittest")
+    sp.call("python -m unittest discover -s . -p '*_test.py'",shell=True)
+    os.chdir(os.getcwd() + "/../../video-deleter/unittest")
+    sp.call("python -m unittest discover -s . -p '*_test.py'",shell=True)
+    os.chdir(os.getcwd() + "/../../video-download/unittest")
+    sp.call("python -m unittest discover -s . -p '*_test.py'",shell=True)
```

Comment:
Det her kører kun, hvis man er Lars.. Jeg skal i hvert fald skrive Python3, for at køre python

Reply:
Ja, så det er vel platform-dependent. Men vil også mene, at python3 er bedre end bare python her

Reply:
scrubs